src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenWinMdEvents.vb

```diff
@@ -896,6 +896,35 @@ End Class
 ]]>)
         End Sub
 
+        <Fact()>
+        Public Sub CustomEventWinMd()
+            Dim source =
+<compilation>
+    <file name="a.vb">
+Imports System.Runtime.InteropServices.WindowsRuntime
+
+Class Test
+    Public Custom Event CustomEvent As System.Action(Of Integer)
+        AddHandler(value As System.Action(Of Integer))
+            Return Nothing		
+        End AddHandler
+
+        RemoveHandler(value As EventRegistrationToken)
+
+        End RemoveHandler
+
+        RaiseEvent()
+
+        End RaiseEvent
+    End Event
+End Class
+    </file>
+</compilation>
+
+            Dim comp = CreateCompilationWithReferences(source, WinRtRefs, options:=TestOptions.DebugWinMD)
+            Dim verifier = CompileAndVerify(comp)
```

Comment:
Consider adding verification that emitted PDB information makes sense. @tmat Should be able to help with that.